### DISCUSSION TITLE
Updated 1 hour ago?

### DISCUSSION THREAD

**Discussion by AbnormalPoof** [FunkinCrew/Funkin#1946] [Q&A] [answered]
I hovered over the repo and it says it was updated 1 hour ago.
Is this a bug or is this the secret repo muffin was talking about????
![Screenshot 2022-02-03 11 37 52 AM](https://user-images.githubusercontent.com/86753001/152397909-7286fca2-23f0-4e5c-b2d7-91c3834ffa65.png)

**Comment by ACrazyTown** [comment]
The secret repo is seperate from the actual one
i think the updated does not actually mean updated as in commits but rather issues/prs opened and that stuff

**Comment by ninjamuffin99** [comment, marked as answer]
the secret repo is a completely seperate repo from this one. Github doesnt allow private branches!!!
